packages/kitchen/src/components/Select/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import styled from "styled-components";
+import { KitchenComponent, NormalSizes } from "../../types";
+
+export type SelectProps = KitchenComponent & {
+  /**
+   * The size of the select.
+   * @default "normal"
+   */
+  size?: NormalSizes;
+  /**
+   * The label of the select.
+   */
+  label?: string;
+  /**
+   * The placeholder of the select.
+   */
+  placeholder?: string;
```

Comment:
On utilise le camelCase en TS, donc `placeHolder`

Reply:
placeholder is a single word so it's already lowerCamelCase here